examples/src/msl/main.rs

```diff
@@ -1,13 +1,49 @@
-extern crate spirv_cross;
 extern crate examples;
+extern crate spirv_cross;
 use spirv_cross::{msl, spirv};
 use examples::words_from_bytes;
+use std::collections::HashMap;
+use std::default::Default;
 
 fn main() {
     let module = spirv::Module::from_words(words_from_bytes(include_bytes!("../vertex.spv")));
 
     // Parse a SPIR-V module
-    let ast = spirv::Ast::<msl::Target>::parse(&module).unwrap();
+    let mut ast = spirv::Ast::<msl::Target>::parse(&module).unwrap();
+
+    // Set some overrides
+    let mut resource_binding_overrides = HashMap::new();
+    resource_binding_overrides.insert(
+        msl::ResourceBindingLocation {
+            stage: spirv::ExecutionModel::Vertex,
+            desc_set: 0,
+            binding: 0,
+        },
+        msl::ResourceBinding {
+            resource_id: 5,
+            force_used: false,
+        },
+    );
+
+    let mut vertex_attribute_overrides = HashMap::new();
+    vertex_attribute_overrides.insert(
+        msl::VertexAttributeLocation(1),
+        msl::VertexAttribute {
+            buffer_id: 1,
+            offset: 2,
+            stride: 3,
+            step: spirv::VertexAttributeStep::Instance,
+            force_used: true,
+        },
+    );
+
+    let compiler_options = msl::CompilerOptions {
+        resource_binding_overrides: Some(resource_binding_overrides),
```

Comment:
I don't think `Option` is useful here

Reply:
Probably not, it makes it harder to use too. I'll switch back to empty hashmaps.